<?xml version="1.0" encoding="UTF-8"?>
<launch>
   <!-- Launch the RoboterNode -->
   <node name="RoboterNode" pkg="core" type="RoboterNode" output="screen" launch-prefix="gdb -ex run --args" />
</launch>